src/Mantasflowers.Persistence/Authentication/AuthError.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FirebaseAdmin.Auth;
+
+namespace Mantasflowers.Persistence.Authentication
```

Comment:
I would say that Firebase is a `Service`. Persistence project is mostly used for data access layer (e.g., database access, ORM (efcore). etc....). Then httpClients that you are injecting in the controller could be moved to be serviceAgents (see other comments).